### PR TITLE
refactor(experimental): update package list in library README

### PR DESCRIPTION
**Pull request by lorisleiva**

### CODE REVIEW

**Review by steveluscher — APPROVED** [review]
Added via Giphy

File: packages/library/README.md [hunk 1]
```diff
@@ -110,10 +110,17 @@ The 2.0 library is tree-shakable, and that tree-shakeability is enforced in the
 
 The new library itself is comprised of several smaller, modular packages under the `@solana` organization, including:
 
--   `@solana/rpc-transport`: For building and managing RPC transports
--   `@solana/rpc-core`: The type-spec of the Solana JSON RPC
+-   `@solana/accounts`: For fetching and decoding accounts
+-   `@solana/codecs`: For building data (de)serializers
+-   `@solana/errors`: For catching and handling errors with minimal production bundle size
+-   `@solana/rpc`: For building and managing RPC calls
```
[the remark this answers is not in the document]
Comment (lorisleiva) [reply packages/library/README.md:116]: Used:

```md
-   `@solana/rpc`: For sending RPC requests
-   `@solana/rpc-subscriptions`: For subscribing to RPC notifications
```

File: packages/library/README.md [hunk 2]
```diff
@@ -110,10 +110,17 @@ The 2.0 library is tree-shakable, and that tree-shakeability is enforced in the
 
 The new library itself is comprised of several smaller, modular packages under the `@solana` organization, including:
 
--   `@solana/rpc-transport`: For building and managing RPC transports
--   `@solana/rpc-core`: The type-spec of the Solana JSON RPC
+-   `@solana/accounts`: For fetching and decoding accounts
+-   `@solana/codecs`: For building data (de)serializers
```
[the remark this answers is not in the document]
Comment (lorisleiva) [reply packages/library/README.md:114]: Used: _For composing data (de)serializers from a set of primitives or building custom ones_

**Comment by lorisleiva** [comment]
### Merge activity

* **Mar 19, 6:06 AM EDT**: @lorisleiva started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/solana-labs/solana-web3.js/2335).

**Pull request merged**